### PR TITLE
Prod

### PR DESCRIPTION
**Pull request by aparnakhadatkar**
# Module: 00
## Lab/Demo: 01, 03. 04. 13. 14

Fixes # .

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: Instructions/Labs/LAB[MB-240]_Lab03_Resource_Configuration.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ In this exercise, you will create the location where resources will start and en
 
 1. Select **User (1)** from the **Resource Type** drop-down field.
 
-1. Select the user record you are signed in as ex. **ODL User [DeploymentID]** for the **User (2)**.
+1. Select the user record you are signed in as ex. **ODL_User [DeploymentID]** for the **User (2)**.
```
Comment (akn9050) [inline Instructions/Labs/LAB[MB-240]_Lab03_Resource_Configuration.md:73]: @aparnakhadatkar @junnhssn , we were planning to not include Deployment ID can you check

Comment (aparnakhadatkar) [reply Instructions/Labs/LAB[MB-240]_Lab03_Resource_Configuration.md:73]: Sure @akn9050

Comment (aparnakhadatkar) [reply Instructions/Labs/LAB[MB-240]_Lab03_Resource_Configuration.md:73]: In Some cases, we have to select the username  @akn9050   so we updated it like this.

Comment (junnhssn) [reply Instructions/Labs/LAB[MB-240]_Lab03_Resource_Configuration.md]: Amit, We are using this when we are asking the user to select the ODL user as part of the lab.

**Pull request merged**